presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -123,6 +124,12 @@ public SystemSessionProperties(
                         "Compute hash codes for distribution, joins, and aggregations early in query plan",
                         featuresConfig.isOptimizeHashGeneration(),
                         false),
+                booleanSessionProperty(
+                        DISTRIBUTED_JOIN,
+                        "Use a distributed join instead of a broadcast join",
+                        featuresConfig.isDistributedJoinsEnabled(),
+                        false),
```

Comment:
hidden: `true` ?

Reply:
I thought about that, but I was worried if people were using it, it would be confusing not to see how it was set.  Not sure what's the better approach though.  happy to change it.

Reply:
fine, leave as is. Just change description, mentioning the new special role of this property.